src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingVisitor.cs

```diff
@@ -141,6 +142,18 @@ public override void VisitMarkupBlock(MarkupBlockSyntax node)
 
         public override void VisitMarkupElement(MarkupElementSyntax node)
         {
+            // If this is the direct child of a @section directive then we add a Razor indent to essentially
+            // set the baseline one level in.
+            // Note: "Direct child" in a little poetic license, as the structure is actually:
+            //     MarkupElement -> MarkupBlock -> CSharpCodeBlock (¯\_(ツ)_/¯) -> RazorDirectiveBody -> RazorDirective
```

Comment:
lmao @javiercn here's an example of what we have to do in order to make sense of the Razor syntax tree for something as simple as an `@section` directive. aka related to https://github.com/dotnet/aspnetcore/issues/34957